packages/cstruct/cstruct.2.4.1/opam

```diff
@@ -0,0 +1,61 @@
+opam-version: "1.2"
+maintainer:   "anil@recoil.org"
+authors:      ["Anil Madhavapeddy" "Richard Mortier" "Thomas Gazagnaire"
+               "Pierre Chambart" "David Kaloper" "Jeremy Yallop" "David Scott"
+               "Mindy Preston" ]
+homepage:     "https://github.com/mirage/ocaml-cstruct"
+license:      "ISC"
+dev-repo:     "https://github.com/mirage/ocaml-cstruct.git"
+bug-reports:  "https://github.com/mirage/ocaml-cstruct/issues"
+tags: [
+  "org:mirage"
+  "org:ocamllabs"
+  "org:xapi-project"
+]
+build: [
+  ["./configure"
+      "--prefix" prefix
+      "--%{lwt+base-unix:enable}%-lwt"
+      "--enable-ppx"
+      "--%{async:enable}%-async"
+      "--%{base-unix:enable}%-unix"]
+  [make]
+]
+build-test: [
+  ["./configure"
+      "--prefix" prefix
+      "--%{lwt:enable}%-lwt"
+      "--enable-ppx"
+      "--%{async:enable}%-async"
+      "--%{base-unix:enable}%-unix"
+      "--enable-tests"]
+  [make]
+  [make "test"]
+]
+install: [
+  [make "install"]
+  [make "js-install"]
+]
+remove:  [
+  [make "js-uninstall"]
+  ["ocamlfind" "remove" "cstruct"]
+]
+depends: [
+  "ocamlfind" {build}
+  "ounit"     {test}
+  "ocplib-endian"
+  "sexplib"
+  "base-bytes"
+  "ocaml-migrate-parsetree"
```

Comment:
@avsm @let-def does it needs a version constraint to be compatible with https://github.com/ocaml/opam-repository/pull/9029 (not sure why that other PR requires this one to be merged first)